pandas/plotting/compat.py

```diff
@@ -0,0 +1,51 @@
+# being a bit too dynamic
+# pylint: disable=E1101
+from __future__ import division
+
+from distutils.version import LooseVersion
+
+
+def _mpl_le_1_2_1():
+    try:
+        import matplotlib as mpl
+        return (str(mpl.__version__) <= LooseVersion('1.2.1') and
+                str(mpl.__version__)[0] != '0')
```

Comment:
I had to add these checks for `'0'` when matplotlib's versioneer was broken for 1.5. I believe it's fixed now. Do you want to remove them in here, or a separate PR?